tools/internal_ci/linux/grpc_xds_k8s_xlang.sh

```diff
@@ -60,7 +60,6 @@ run_test() {
     --xml_output_file="${TEST_XML_OUTPUT_DIR}/${tag}/${clang}-${slang}/sponge_log.xml" \
     --force_cleanup \
     --nocheck_local_certs
-  set +x
```

Comment:
Good catch. I remember it confused me too, but I completely forgot this needs to be removed to support `failed_tests` counter.